docs/backend-requests/resources/session-tokens.mdx

```diff
@@ -1,42 +1,40 @@
 ---
 title: Session tokens
-description: When a user is authenticated in your application, a short-lived session token is generated by Clerk.js that depicts the fact and it's sent to your backend. Your backend will typically want to validate that the session token is valid (i.e. that it comes from Clerk, that it hasn't expired etc.)
+description: Learn about session tokens and how to validate them in your backend.
 ---
 
 # Session tokens
 
-When a user is authenticated in your application, a short-lived session token is generated by Clerk.js that depicts the fact and it's sent to your backend. Your backend will typically want to validate that the session token is valid (i.e. that it comes from Clerk, that it hasn't expired etc.)
+When a user is authenticated in your application, Clerk generates a short-lived session token that you can use to authenticate requests to your backend. This token is a JSON Web Token (JWT) that contains information about the user and their session.
 
 ## Default session claims
 
-Below are the default session claims that Clerk generates for you:
+In every generated token, there are certain claims that are automatically included and cannot be overridden by templates. Clerk calls these "default claims". Many of these are standard claims registered by IANA. For generic information about them, refer to [RFC 7519](https://datatracker.ietf.org/doc/html/rfc7519#section-4.1).
```

Comment:
```suggestion
Every generated token has default claims that cannot be overridden by templates. Clerk's default claims include:
```
Reduced words, and moved the next line into this paragraph. In the next suggestions I'll add links to the relevant section of the RFC for each claim